Automattic/woocommerce-payments
# Fix: align table items in screens: Deposits, Transactions, and Disputes

Fixes #2351

#### Changes proposed in this Pull Request

- I just simply add this property to columns that we'd aligin left

```js
			isLeftAligned: true
``` 

- Remove some CSS code that has aligned these items to `center`. 
- Update relevant snapshots. 

#### Testing instructions

1. Visit these pages under Deposits, Transactions, and Disputes, and notice their layouts.
2. Apply this PR.
3. Remember to run `npm run dev` to rebuild the React code. 
4. Visit these pages again. 
5. Expect: **Text fields should be left-aligned and numeric values should be right-aligned.** 

### Screenshots - left is the current `develop` branch, right is with this PR. 

![dep](https://user-images.githubusercontent.com/10045087/127281532-7f705730-2b7c-4002-8bc2-fb28920aea3f.png)

![trans](https://user-images.githubusercontent.com/10045087/127281541-41af6ee7-9cd2-4005-a85d-70532f083785.png)

![dis](https://user-images.githubusercontent.com/10045087/127281560-54d9d956-6e92-499b-a091-5e860fadc191.png)





-------------------
